challenge-weather-app/index.html

```diff
@@ -1,49 +1,117 @@
-        <input class="search__input" id="search-tf" name="city" placeholder="Enter city name" autocomplete="city" />
-        <button class="btn search__btn">Go</button>
-      </form>
-    </div>
-  </main>
-
-  <!-- JS goes here -->
-</body>
+    <title>Meteoropolis</title>
+    <link rel="stylesheet" href="assets/globals.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+  </head>
 
+  <body>
+    <!-- Structure -->
+    <main class="content">
+      <header class="header">
+        <h1 class="title">
+          <i>Meteor</i>
+          <i>opolis</i>
+        </h1>
+      </header>
+
+      <figure class="photo" id="photo"></figure>
+
+      <div class="info">
+        <p class="info__item info__item--conditions" id="conditions"></p>
+        <p class="info__item info__item--credits">
+          <a href="#" id="credit-user"></a>
+          <span>on</span>
+          <a href="#" id="credit-platform">Unsplash</a>
+        </p>
+      </div>
+
+      <div class="thumbs" id="thumbs"></div>
+
+      <div class="controls">
+        <form class="search" id="search">
+          <label class="search__label" for="search-tf">City</label>
+          <input class="search__input" id="search-tf" name="city" placeholder="Enter city name" autocomplete="city" />
+          <button class="btn search__btn">Go</button>
+        </form>
+      </div>
+    </main>
+
+    <!-- JS goes here -->
+
+    <script>
+      const apiKey = "2d2afc913f90548905b086803a05ef5d";
+      const accessKey = "m89STg41g63uXj4rt_9u5GXtN9LTGDH_ViVVX9Sy9Yc";
+      const searchInput = document.getElementById("search-tf");
+      const thumbCont = document.getElementById("thumbs");
+      let linkCont = {};
+      const figure = document.getElementById("photo");
+
+      document.getElementById("search").addEventListener("submit", (event) => {
+        event.preventDefault();
+        getFetchCity(searchInput.value, apiKey);
+      });
+
+      function getFetchCity(input, key) {
+        fetch(`https://api.openweathermap.org/data/2.5/weather?q=${input}&APPID=${key}`)
+          .then((response) => response.json())
+          .then((data) =>
+            data.weather[0].description ? getFetchPhoto(data.weather[0].description) : console.log("no city")
+          );
+      }
+      function getFetchPhoto(input) {
+        fetch(`https://api.unsplash.com/search/photos?query=${input}&client_id=${accessKey}`)
+          .then((response) => response.json())
+          .then((data) => {
+            generateThumbs(data.results);
+            getMainPhotoLinks(data.results);
+            generateMainPhoto(linkCont);
+            thumbCont.addEventListener("click", (event) => {
+              const imgId = event.target.id;
+              generateMainPhoto(linkCont, imgId);
+            });
+          });
+      }
+
+      function generateThumbs(photos) {
+        thumbCont.innerHTML = "";
+        photos.map((photo) => {
```

Comment:
This is a little tricky. If you don't need to return data from your loop, you can use `forEach` instead of `map`. Use `map` when you want to do some calculation and return data.